src/tcp_listener_sup.erl

```diff
@@ -49,10 +49,11 @@ start_link(IPAddress, Port, Transport, SocketOpts, ProtoSup, ProtoOpts, OnStartu
 init({IPAddress, Port, Transport, SocketOpts, ProtoSup, ProtoOpts, OnStartup, OnShutdown,
       ConcurrentAcceptorCount, Label}) ->
     {ok, AckTimeout} = application:get_env(rabbit, ssl_handshake_timeout),
+    MaxConnections = application:get_env(rabbit, max_connections, infinity),
```

Comment:
`application:get_env/3` isn't available in Erlang/OTP R16B03. Please use `rabbit_misc:get_env/3`.